packages/react-devtools-shared/src/devtools/views/Components/InspectedElement.js

```diff
@@ -92,15 +92,46 @@ export default function InspectedElementWrapper(_: Props) {
     (canViewElementSourceFunction === null ||
       canViewElementSourceFunction(inspectedElement));
 
+  const isErrored = inspectedElement != null && inspectedElement.isErrored;
+  const targetErrorBoundaryID =
+    inspectedElement != null ? inspectedElement.targetErrorBoundaryID : null;
+
   const isSuspended =
     element !== null &&
     element.type === ElementTypeSuspense &&
     inspectedElement != null &&
     inspectedElement.state != null;
 
+  const canToggleError =
+    inspectedElement != null && inspectedElement.canToggleError;
+
   const canToggleSuspense =
     inspectedElement != null && inspectedElement.canToggleSuspense;
 
+  const toggleErrored = useCallback(() => {
+    if (inspectedElement == null || targetErrorBoundaryID == null) {
+      return;
+    }
+
+    if (targetErrorBoundaryID !== inspectedElement.id) {
+      dispatch({
+        type: 'SELECT_ELEMENT_BY_ID',
+        payload: targetErrorBoundaryID,
+      });
+    }
+
+    const rendererID = store.getRendererIDForElement(targetErrorBoundaryID);
+
+    // Toggle error.
+    if (rendererID !== null) {
+      bridge.send('overrideError', {
+        id: targetErrorBoundaryID,
+        rendererID,
+        forceError: !isErrored,
+      });
+    }
```

Comment:
Oooh, I didn't notice this earlier, and it's just a small nit, but let's maybe wrap both actions in this conditional since it doesn't make sense to do the one unless we're going to also do the other. 😁  (We should never _really_ have a null `rendererID` anyway, just a precaution.)

```suggestion
    const rendererID = store.getRendererIDForElement(targetErrorBoundaryID);
    if (rendererID !== null) {
      if (targetErrorBoundaryID !== inspectedElement.id) {
        // Update tree selection so that if we cause a component to error,
        // the nearest error boundary will become the newly selected thing.
        dispatch({
          type: 'SELECT_ELEMENT_BY_ID',
          payload: targetErrorBoundaryID,
        });
      }

      // Toggle error.
      bridge.send('overrideError', {
        id: targetErrorBoundaryID,
        rendererID,
        forceError: !isErrored,
      });
    }
```